src/Components/ClusterRules/ClusterRules.spec.ct.js

```diff
@@ -64,7 +64,7 @@ const filtersConf = {
   },
 };
 
-const filterData = (data, filters) => filter(filtersConf, data, filters);
+const filterData = (filters) => filter(filtersConf, data, filters);
```

Comment:
I think here you pass `data` by reference, not a value (objects and arrays in JS passed by reference in functions). It didn't lead to problems in tests. However, it may create issues in the future...

Reply:
That's why I didn't put too much attention to this. However, it might help to avoid some troubles, so I will change it.

Reply:
@gkarat Do you think it is better now to have `values` as `let` and `data` as `const`? Is it more clear?